### DISCUSSION TITLE
It is possible to add several images to one entity?

### DISCUSSION THREAD

**Discussion by developer-16** [philomena-dev/philomena#264] [Q&A]
Hey!

Right now one entity in philomena (like this one in derpibooru: https://derpibooru.org/images/3375264) corresponds to exactly one image, gif or video. Is it possible for it to contain several images instead? I want to use philomena as a mushroom identification tool, so that user would be able to find all the mushrooms matching the tags that they've entered. That means that each entry would match one mushroom species instead of one mushroom image. And it would actually be very useful to have several images present for each species. 

I know that this is not exactly what philomena was made for, but I will be grateful for any advice :)

**Comment by liamwhite** [comment]
I do not really think this is feasible given the current design. The idea is that image models are 1:1 with uploads. I would suggest using multiple images for this purpose.
